src/main/webapp/app/entities/flair-bi/modal/modal-tabs/condition/condition.component.js

```diff
@@ -51,7 +51,9 @@
         }
 
         function getMetadataTooltip(metadata) {
-            return 'from ' + metadata.startDateFormatted + ' to ' + metadata.endDateFormatted;
+            if(metadata){
+                return 'from ' + metadata.startDateFormatted + ' to ' + metadata.endDateFormatted;
+            }
```

Comment:
a function should always return a value, so perhaps you can add `return ''` in the bottom

Reply:
okay